renovate.json

```diff
@@ -4,7 +4,9 @@
     {
       "matchUpdateTypes": ["minor", "patch"],
       "matchPackagePatterns": ["*"],
-      "automerge": true
+      "automerge": true,
+      "stabilityDays": 7,
+      "internalChecksFilter": "strict"
```

Comment:
Following recommendation from: https://docs.renovatebot.com/configuration-options/#internalchecksfilter (although I'm not enabling the dependencyDashboard)
> All pending releases will be filtered. PRs will be skipped unless a non-pending version is available

Reply:
> Setting dependencyDashboard ([docs](https://docs.renovatebot.com/configuration-options/#dependencydashboard)) to true will create a special “Dependency Dashboard” issue, whose description text lists all PRs created by the bot in one place (where PRs can have any state, including pending, open, closed, or error). It contains clickable links, e.g. to rebase/retry multiple PRs without having to open each one individually. I recommend you read the docs of dependencyDashboard and its sub-options (such as dependencyDashboardApproval) to understand how it works.

In case we want to consider this

Reply:
Yeah, that's a good point, actually. Getting the bot updates off the GH board would actually be realllly nice 😅